guides/md/serialization_and_saving.md

```diff
@@ -134,9 +141,20 @@ reconstructed_model.fit(test_input, test_target)
 
 <div class="k-default-codeblock">
 ```
-4/4 [==============================] - 0s 833us/step - loss: 0.2464
+2021-11-09 15:11:59.097804: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
+2021-11-09 15:11:59.097859: W tensorflow/stream_executor/cuda/cuda_driver.cc:269] failed call to cuInit: UNKNOWN ERROR (303)
+2021-11-09 15:11:59.097888: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (penguin): /proc/driver/nvidia/version does not exist
+2021-11-09 15:11:59.098145: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
+To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
+
+4/4 [==============================] - 0s 2ms/step - loss: 0.3577
 
-<tensorflow.python.keras.callbacks.History at 0x1511b87d0>
+2021-11-09 15:11:59.631545: W tensorflow/python/util/util.cc:368] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
```

Comment:
Make sure the md file doesn't include any warnings / log lines this these

Reply:
Do we manually remove them after running autogen, or is there a setting that excludes the log lines?